.github/workflows/verify_api.yml

```diff
@@ -2,23 +2,21 @@ name: Verify API
 on:
   push:
     branches:
-      - master
+      - main
       - release-*
     paths:
       - 'scripts/*'
       - 'api-generator/*'
   pull_request:
     branches:
-      - master
+      - main
       - release-*
     paths:
       - 'scripts/**'
       - 'api-generator/**'
 jobs:
   verify:
     timeout-minutes: 30
-    strategy:
```

Comment:
drive-by: only relevant if a matrix gets used. In this case there isn't used any.